Mathlib/Data/Fin/Tuple/Basic.lean

```diff
@@ -410,6 +411,41 @@ theorem append_castAdd_natAdd {f : Fin (m + n) → α} :
   unfold append addCases
   simp
 
+theorem append_injective_iff {α : Type*} {m n : ℕ} {x : Fin m → α} {y : Fin n → α} :
+  Function.Injective (Fin.append x y) ↔
+    Function.Injective x ∧ Function.Injective y ∧ Disjoint (Set.range x) (Set.range y) := by
```

Comment:
```suggestion
    Function.Injective (Fin.append x y) ↔
      Function.Injective x ∧ Function.Injective y ∧ Disjoint (Set.range x) (Set.range y) := by
```
since the statement should be indented four spaces.